README.md

```diff
@@ -1,13 +1,15 @@
-# 5 Calls React Port
+# 5calls
 
 [![CircleCI](https://circleci.com/gh/5calls/react-dev.svg?style=svg)](https://circleci.com/gh/5calls/react-dev)
 
-This repo holds code for the port of the [5 Calls web site](https://5calls.org) from the [Choo](https://choo.io) front-end framework to [React](https://facebook.github.io/react/) with [Redux](http://redux.js.org/) for state management and [Typescript](https://www.typescriptlang.org/) for type safety and documentation.
+This repo holds the current frontend for the [5 Calls web site](https://5calls.org). Please check the [issue list](https://github.com/5calls/5calls/issues), pull requests and contributor guidelines before starting work so we can ensure you're not duplicating work! We're all volunteers and want to treat the time you dedicate to the site with respect. Ping [@make5calls](https://twitter.com/make5calls) on Twitter with your email to get an invite to our Slack.
```

Comment:
Please change [issues list] to [issues list on the 5calls/5calls repo] to clarify that we are referring to the other repo.